doc/source/example_notebooks/IS2_data_visualization.ipynb

```diff
@@ -7,7 +7,13 @@
    "source": [
```

Comment:
<p>The whole notebook isn't non-functional. It should work up until the "Visualize ICESat-2 elevation using OpenAltimetry API" header.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icesat2py/icepyx/pull/540/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I meant to communicate more that it's not going to get to the intended point due to the error, so I thought it made sense to callout up-front "this doc may not take you where you want to go". What do you think about how to adjust the wording?

Reply:
<p>That makes sense - and I like the callout happening at the start. I like how you wrote it if you read it immediately after the notebook description, but I skipped right to the "Danger" part (so it's working as intended and got my attention!). How about something like: "Some of this notebook is currently non-functional"?</p>

Reply:
Sounds good! :rocket:  Thanks Jessica!

Reply:
9c9e722